README.md

```diff
@@ -108,47 +72,74 @@ With this technology we want to **achieve**:
    npm run dev
    ```
 
+<br/>
+
+## 💻 Technology Used
+
+Qiskit.org is a static website generated using [Nuxt](https://nuxt.com/).
+
+We create and run tests using [Vitest](https://vitest.dev/), avoid syntax errors using [ESLint](https://eslint.org/) and [Stylelint](https://stylelint.io/), and automate code integration and deployment using [GitHub Actions](https://github.com/features/actions).
+
+With these technologies we aim to achieve the following:
+
+- Using a component-based framework (like Vue) that allows us to reuse the UI code in different parts of the application.
+- Fast initial page load.
+- Index content on Search Engines.
+- Test JS unit functions.
+- Avoid syntax errors.
+- Continuous integration pipeline.
+
```

Comment:
This list doesn't feel that relevant to me. I believe that it's quite self-explanatory given the tech-stack listed above. It doesn't offer much value to a contributor.

Reply:
Removed in https://github.com/Qiskit/qiskit.org/pull/3172/commits/e663ede4b578cbd2c4034f7a29d7e5b5588449d1